### DISCUSSION TITLE
Feature request: Enable reordering the quick links by dnd

### DISCUSSION THREAD

**Discussion by hiergiltdiestfu** [bluecaret/carettab#332] [Ideas and Feature Requests]
In the v3 Beta, the only way to reorder the quick links is currently to use the up and down arrows next to the text boxes. This is tiresome for long link lists. 

I'd like to be able to reorder them by drag and drop instead.

![image](https://user-images.githubusercontent.com/10924590/50869174-36255200-13b3-11e9-9641-5330af23c437.png)

**Comment by bluecaret** [comment]
Drag and drop will be in the next update :)
